yt/yt/server/lib/exec_node/config.cpp

```diff
@@ -109,15 +109,19 @@ void TCriJobEnvironmentConfig::Register(TRegistrar registrar)
 
 void TSlotLocationConfig::Register(TRegistrar registrar)
 {
-    registrar.Parameter("disk_quota", &TThis::DiskQuota)
+    registrar.Parameter("disk_quota", &TThis::MaxDiskSpace)
```

Comment:
Let name it `max_disk_space` in this case and add an alias for `disk_quota`

Reply:
I'll drop this renaming.